src/commands/platforms/destroy.ts

```diff
@@ -63,7 +68,11 @@ export default class PlatformDestroy extends Command {
     const { data: account_platform } = await this.app.api.get(`/accounts/${account.id}/platforms/${platform.name}`);
 
     cli.action.start(chalk.blue('Deregistering platform'));
-    await this.app.api.delete(`/platforms/${account_platform.id}`);
+    await this.app.api.delete(`/platforms/${account_platform.id}`, {
+      params: {
+        force: answers.force ? 1 : 0,
```

Comment:
Only set params if force is set. Prevents breaking changes for on-prem.

```ts
const params = {};
if (answers.force) {
  params.force = 1;
}
```

Reply:
Yikes, good catch! Will do 👍